### DISCUSSION TITLE
Stiped merging

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#246] [Ideas]
The idea is similar to tensor (and I don't know what tensor2 is).
It also uses alpha and beta, but alpha and beta are integers.
Alpha means how many tensors will be taken from modelA (strating from 0) and beta from modelB (strarting from 0+alpha). So alpha and beta are stripes and go one by one to the end of element.

alpha+beta+alpha+beta+...

I'm not sure if it's clear :-(

**Comment by TomLucidor** [comment]
Could you explain more about this? Since this sounds like fragmented addition rather than a hacked version of non-linear averaging.

**Reply by miasik** [reply]
Sure ;-)
Let it be "fragmented addition" as you said. No complex mathematics, just fragments one by one

**Reply by TomLucidor** [reply]
If true then you only need 2 steps for 2 variables? How is this different?

**Reply by miasik** [reply]
> If true then you only need 2 steps for 2 variables? How is this different?

a and b aren't float numbers, they're integer ones. We need to repeat a+b+a+b+... until the end of a block.
